package-lock.json

```diff
@@ -30,10 +30,10 @@
       }
```

Comment:
Hmm, I don't think there should be changes in this file.

Reply:
Fixed